sigle/src/pages-lib/_error.tsx

```diff
@@ -1,77 +1,218 @@
+    pr: '$3',
+  },
+
+  '@xl': {
+    pr: '$3',
+  },
+});
+
+const NotFoundText = styled(Text, {
+  mb: '$6',
+  maxWidth: '400px',
+});
+
+const NotFoundWrapper = styled('div', {
+  width: '300px',
+  height: '282px',
+  position: 'relative',
+  mt: '$4',
+  mb: '$4',
+
+  '@lg': {
+    minWidth: '520px',
+    height: '488px',
+  },
+
+  '@2xl': {
+    width: '737px',
+    height: '693px',
+  },
+});
+
+const ErrorWrapper = styled('div', {
+  width: '300px',
+  height: '232px',
+  maxWidth: '100%',
+  position: 'relative',
+  mt: '$4',
+  mb: '$4',
+
+  '@lg': {
+    width: '520px',
+    height: '403px',
+  },
+
+  '@2xl': {
+    width: '765px',
+    height: '593px',
+  },
+});
+
+const NotFoundIllu = ({
+  statusCode,
+  alt,
+}: {
+  statusCode: number;
+  alt: string;
+}) => {
+  if (statusCode === 404) {
+    return (
+      <Image
+        src="/static/img/404.png"
+        alt={alt}
+        priority
+        layout="fill"
+        objectPosition="center"
+      />
+    );
   }
-`;
+
+  return (
+    <Image
+      src="/static/img/5XX.png"
+      alt={alt}
+      priority
+      layout="fill"
+      objectPosition="center"
+    />
+  );
+};
 
 interface ErrorProps {
   statusCode: number;
   errorMessage?: string | null;
   hasGetInitialPropsRun?: boolean;
   err?: Error;
+  sentryErrorId?: string;
 }
 
 export const MyError = ({
   statusCode,
   hasGetInitialPropsRun,
   errorMessage,
   err,
+  sentryErrorId,
 }: ErrorProps) => {
+  const { user } = useAuth();
+  const notFound = statusCode === 404;
+
   if (!hasGetInitialPropsRun && err) {
     // getInitialProps is not called in case of
     // https://github.com/vercel/next.js/issues/8592. As a workaround, we pass
     // err via _app.js so it can be captured
-    Sentry.captureException(err);
+    sentryErrorId = Sentry.captureException(err);
     // Flushing is not required in this case as it only happens on the client
   }
 
   return (
-    <NotFoundContainer>
-      <NotFoundIllu src="/static/img/jungle.png" alt="One" />
-      <NotFoundTextContainer>
-        <NotFoundTitle data-testid="error-title">
-          {statusCode} in sight!
-        </NotFoundTitle>
-        <NotFoundSubTitle data-testid="error-sub-title">
-          {errorMessage
-            ? errorMessage
-            : 'Looks like you and Julia went too far...'}
-        </NotFoundSubTitle>
-        <Link href="/">
-          <Button as="a">Go to the app</Button>
-        </Link>
-      </NotFoundTextContainer>
-    </NotFoundContainer>
+    <>
+      <AppHeader />
+      <NotFoundContainer>
+        {notFound ? (
+          <NotFoundWrapper>
+            <NotFoundIllu statusCode={404} alt="A lost traveller" />
+          </NotFoundWrapper>
+        ) : (
+          <ErrorWrapper>
+            <NotFoundIllu statusCode={500} alt="Woodpecker and broken pencil" />
+          </ErrorWrapper>
+        )}
+        <NotFoundTextContainer>
+          <Heading
+            css={{ mb: '$3', lineHeight: 1 }}
```

Comment:
why is the line-height different, is it an issue with the design of the page or the values of the current design system?